### PR TITLE
Rewrite About page for grid-website

### PR DESCRIPTION
**Pull request by chenette**
- Expand Grid description to clarify goal of a cross-industry tool set
  Note: Some content is derived from the Hyperledger Grid project page at
  https://www.hyperledger.org/projects/grid

- Add image from Grid proposal

- Add details of component types, with examples (mention GS1, Grid Track and Trace, Pike, and the grid CLI)

- Add "How to participate" paragraph with link to more info in the FAQ (in a separate PR)

- Add "Source code" with summary of grid, grid-contrib, and grid-rfcs repos

Signed-off-by: Anne Chenette <chenette@bitwise.io>

### CODE REVIEW

File: generator/source/about.md [hunk 1]
```diff
@@ -9,37 +9,85 @@ feat_img_size: small
...
-# Hyperledger Grid
+Hyperledger Grid is a project to provide a set of shared, reusable tools for
+building cross-industry supply chain solutions based on distributed ledger
+platforms. Its goal is to simplify the development of distributed-ledger-based
+solutions for all types of supply chain scenarios.
 
-Hyperledger Grid is a project for building supply chain solutions. It includes
-a set of libraries, data models, and SDKs to accelerate development for supply
-chain smart contracts and client interfaces.
+This ecosystem of technologies, frameworks, and libraries will be designed to
```
Comment (jsmitchell) [inline generator/source/about.md]: s/will be/is/

File: generator/source/about.md [hunk 2]
```diff
@@ -9,37 +9,85 @@ feat_img_size: small
...
+work with existing blockchain platform software and business-specific
```
Comment (jsmitchell) [inline generator/source/about.md]: s/blockchain/distributed ledger/

Comment (chenette) [reply generator/source/about.md]: Done

File: generator/source/about.md [hunk 3]
```diff
@@ -9,37 +9,85 @@ feat_img_size: small
...
-This Hyperledger project is in _incubation_. It was proposed to the community
-and documented in this [Hyperledger Grid
-Proposal](https://docs.google.com/document/d/1b6ES0bKUK30E2iZizy3vjVEhPn7IvsW5buDo7nFXBE0/edit).
-Information on what _incubation_ means can be found in the [Hyperledger Project
-Lifecycle document](https://wiki.hyperledger.org/community/project-lifecycle).
+* Domain-specific data models based on existing open standards, such as
+  GS1 product definitions that can be used in a product catalog
 
-# License
+* Reference implementations of supply-chain-centric data types
```
Comment (davececchi) [inline generator/source/about.md]: This is redundant w/ prior bullet.  

I like:

Reference implementations of supply-chain-centric data types, including domain-specific models based on existing open standards, such as GS1.

File: generator/source/about.md [hunk 4]
```diff
@@ -9,37 +9,85 @@ feat_img_size: small
...
 
-Hyperledger Grid software is licensed under the
-[Apache License Version 2.0](https://github.com/hyperledger/grid/blob/master/LICENSE)
-software license.
+* Smart contract business logic based on industry best practices. For example,
```
Comment (davececchi) [inline generator/source/about.md]: - "grounded in existing industry processes and best practices"
- I would drop the stuff about Track and Trace here and move it to the "example applications stuff"
- Pike should be it's own bullet point

File: generator/source/about.md (hunk 3 above)
Comment (chenette) [reply generator/source/about.md]: Rewrote

File: generator/source/about.md (hunk 4 above)
Comment (chenette) [reply generator/source/about.md]: Made these changes

File: generator/source/about.md [hunk 5]
```diff
@@ -9,37 +9,85 @@ feat_img_size: small
...
+  domain-specific data models based on existing open standards such as
+  [GS1](https://www.gs1.org/standards).
 
-# License
+* Smart-contract business logic based on industry best practices.
 
-Hyperledger Grid software is licensed under the
-[Apache License Version 2.0](https://github.com/hyperledger/grid/blob/master/LICENSE)
-software license.
+* Pike, a smart contract that handles organization and identity permissions
```
Comment (davececchi) [inline generator/source/about.md]: Pike, a Sabre based framework that handles...

File: generator/source/about.md (hunk 4 above)
Comment (davececchi) [reply generator/source/about.md]: Pike, a Sabre based framework that handles...

**Comment by davececchi** [comment]
Suggested one more minor change to Pike language but approving as it stands now so as not to appear nit picky.

File: generator/source/about.md (hunk 4 above)
Comment (davececchi) [reply generator/source/about.md]: Pike, a Sabre based framework that handles...

**Comment by chenette** [comment]
Made the suggested change. Thanks!

**Pull request merged**